editor/views/rai.py

```diff
@@ -7,11 +7,16 @@
 from events.rai import handle_rai_change
 from events.rai import RaiEvent
 
+_INFO_TEXT = """This tab is the Responsible AI extension of Croissant. **Filling this tab is optional.**
```

Comment:
This is the text added at the top of the RAI Tab. Please feel free to change it as you want.